plier_recount3.smk

```diff
@@ -0,0 +1,97 @@
+"""
+PLIER recount3
+"""
+
+# ==============================================================================
+# IMPORT LIBS
+# ==============================================================================
+
+from os.path import basename
+from glob import glob
+from types import SimpleNamespace
+import os
+import utils
+import pandas as pd
```

Comment:
Interesting. I've never used snakemake before. Why do you need to import these packages here?